tests/test_job_queue.py

```diff
@@ -0,0 +1,26 @@
+
+"""
+Test class for Job-Queue Class.
+"""
+from c20_server.job_queue import JobQueue
+
+
+def test_add_job_():
```

Comment:
Tests should test ideas, not methods.

Make `test_one_job_added_is_returned_by_get` where you add one job, then get one job and make sure it matches the job you added.

`test_add_two_jobs_then_remove_them` where you add two, and then get them back in a FIFO order.

In both tests, ensure that the `get_num_unassigned_jobs` (note spelling change) returns the right number.

Finally, what should happen if someone calls `get_job` when there are no jobs?  I would say raise an exception.